Sources/Utility/Git.swift

```diff
@@ -59,14 +59,34 @@ public class Git {
             let out = (try? Git.runPopen([Git.tool, "-C", repo.path.asString, "tag", "-l"])) ?? ""
             let tags = out.characters.split(separator: "\n").map{ String($0) }
 
-            // First try the plain init.
+            // First, check if we need to restrict the tag set to version-specific tags.
             var knownVersions: [Version: String] = [:]
+            for versionSpecificKey in Versioning.currentVersionSpecificKeys {
+                for tag in tags {
+                    if tag.hasSuffix(versionSpecificKey) {
+                        let specifier = String(tag.characters.dropLast(versionSpecificKey.characters.count))
+                        if let version = Version(specifier) ?? Version.vprefix(specifier) {
```

Comment:
Just historical... the old code was maybe intentionally trying to ignore any v-prefixed versions if no non-prefixed versions are found, but I think that is a misfeature.